IRSA-README.md

```diff
@@ -0,0 +1,88 @@
+# Domino Advanced Credential Propagation for AWS based on IAM Role for Service Accounts (IRSA)
+
+IAM Role for Service Accounts ([IRSA](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html)) is a capability which allows Kubernetes Service Accounts attached to a EKS workload to assume IAM Role(s). In much the same way as you can attach and IAM Role to an EC2 instance at startup.
+
+## Article Topics
+ - Security 
+ - Identity And Access Management (IAM) 
+ - Role Based Access Control (RBAC)
+ - OpenID Connect (OIDC), Web Identity and Attribute Based Access Control (ABAC) 
+
+## High Level Overiew
+
+Domino Data Lab’s platform is built on top of the Kubernetes framework. Domino Nexus is a comprehensive platform that 
+allows you to execute Data Science and Machine Learning workloads across any compute cluster - whether in the cloud, 
+a specific region, or on-premises. By unifying data science silos across the enterprise, Domino Nexus provides a 
+centralized hub for building, deploying, and monitoring models. 
+
+Such a hybrid architecture enables a wide variety of use-cases. For example, you could perform data-processing in the
+AWS cloud and store a training dataset in S3. Next you could trigger a ML Training job on a GPU on an on-premise 
+data plane for better cost performance. How should the on-premise data plane authenticate to AWS IAM to read data 
+from S3? Can the user assume an IAM role transparently based on policy defined in agreement between the Domino
+and the AWS Administration teams?
+
+It is also possible that two teams working on separate Domino data planes hosted on different Cloud Providers in the
+same organization may need to consume resources from each other. Data planes hosted in each of these Cloud Providers 
+may need to assume identities in the other Cloud Providers.  
+
+Using Web Identities based on an OpenID Connect Provider supports such use-cases. In this article we will start with a 
+basic setup where using IAM Role for Service Accounts (IRSA) we can configure Kubernetes Service Accounts attached to 
+an EKS workload to assume IAM Role(s). This is similar to attaching an IAM Role to an EC2 instance at startup. 
+Note that this mechanism allows EKS workloads to assume roles in any AWS Account and not just the AWS account hosting 
```

Comment:
Lets throw in the 'cross-account IAM roles' terminology here?

Reply:
Good idea I will rephrase it as - "This mechanism allows EKS workloads to assume IAM roles across AWS Accounts. You are not limited to assuming an IAM role in the AWS Account hosting the EKS cluster. Eventually through the series of articles we will discover how any K8s cluster hosted on premise or any cloud provider can assuming an IAM role in an AWS Account using this mechanism"